examples/fitting-with-pints.ipynb

```diff
@@ -337,7 +337,7 @@
     "    labels=labels,\n",
     "    trajectory=trajectory,\n",
     "    points=points,\n",
-    "    downsampling=27,\n",
+    "    zoom=0.03,\n",
```

Comment:
```suggestion
    "    zoom=1 / 27,\n",
```

Reply:
Would it make sense to specify the default zoom value like this?

Reply:
I suppose so!